room_alignment/domain.py

```
@@ -475,44 +496,129 @@ def alignment_summary(
     counts = {state: 0 for state in sorted(ALIGNMENT_STATES)}
     audio_confirmed = 0
     timestamp_only = 0
-    conflicts: list[dict[str, Any]] = []
+    eligibility_counts = {state: 0 for state in sorted(PROGRAM_ELIGIBILITY_STATES)}
```

Ruff (0.16.2)

[warning] Unnecessary dict comprehension for iterable; use `dict.fromkeys` instead

Replace with `dict.fromkeys(iterable)`)

(C420)